app/src/main/java/org/oppia/android/app/options/AudioLanguageActivityPresenter.kt

```diff
@@ -13,6 +14,14 @@ class AudioLanguageActivityPresenter @Inject constructor(private val activity: A
 
   fun handleOnCreate(prefKey: String, prefValue: String) {
     activity.setContentView(R.layout.audio_language_activity)
+    val toolbar: androidx.appcompat.widget.Toolbar = activity.findViewById(R.id.audio_language_toolbar)
+    toolbar.setNavigationOnClickListener {
```

Comment:
What happens if user goes back by using android back button instead of app-back-button.

Reply:
@rt4914 I tried this use case it worse fine even if we use android back button instead of app-back button

Reply:
Thanks for verifying @bkaur-bkj. Happy to know that it is working fine :)